jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClient.java

```diff
@@ -115,6 +115,11 @@
             builder.setHostnameVerifier(new TrustAllHostnameVerifier());
         }
 
+        // uses SASL Mechanisms ANONYMOUS and PLAIN, but tries to authenticate with GSSAPI when it's offered
```

Comment:
Can you clarify the comment? We're disabling it because it doesn't work, right?

Reply:
Is it better now?
The previous was just a copy of the one from jicofo